data/core.yaml

```
@@ -346,6 +346,13 @@ en:
       not_downloaded:
         single: This feature can't be moved because parts of it have not yet been downloaded.
         multiple: These features can't be moved because parts of them have not yet been downloaded.
+    follow:
+      key: F
+      error:
+        intersection_of_mutiple_ways: The last two nodes are intersections of multiple ways
+        intersection_of_different_ways: The last two nodes must be part of the same way
+        needs_more_initial_nodes: Draw at least two nodes in the same way first
+        unknown: Failed to continue way
```

Explicitly state that the “follow” operation is disabled and suggest a recovery method:

```suggestion
        intersection_of_mutiple_ways: This line can't follow a way because multiple ways are connected to the line's last two points. Add another vertex manually to continue.
        intersection_of_different_ways: This line can't follow a way because the line is only connected to the way at a single point. Add another point manually to continue.
        needs_more_initial_nodes: This line can't follow a way because it isn't connected to enough consecutive points along a way. Add another point manually to continue.
        unknown: This line can't follow a way.
```